libraries/RC_Channel/RC_Channel.cpp

```diff
@@ -564,6 +565,7 @@ const RC_Channel::LookupTable RC_Channel::lookuptable[] = {
     { AUX_FUNC::CAM_MODE_TOGGLE,"CamModeToggle"},
     { AUX_FUNC::GENERATOR,"Generator"},
     { AUX_FUNC::ARSPD_CALIBRATE,"Calibrate Airspeed"},
+    { AUX_FUNC::RELOCATE_MISSION,"Relocate Mission"},
```

Comment:
I think we probably don't want this, it will result in getting `Relocate Mission: high/low/mid`  messages when the switch is not actually doing anything.